src/main/java/kokhanevich/spring/dao/impl/RentDaoImpl.java

```diff
@@ -0,0 +1,50 @@
+package kokhanevich.spring.dao.impl;
+
+import java.util.List;
+import javax.persistence.TypedQuery;
+import kokhanevich.spring.dao.RentDao;
+import kokhanevich.spring.entity.Book;
+import kokhanevich.spring.entity.Rent;
+import kokhanevich.spring.entity.User;
+import org.hibernate.SessionFactory;
+import org.hibernate.query.Query;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class RentDaoImpl implements RentDao {
+
+    @Autowired
+    private SessionFactory sessionFactory;
+
+    @Override
+    public Rent rentBook(User user, Book book) {
+        Rent rent = new Rent(user, book);
+        sessionFactory.getCurrentSession().save(rent);
+        return rent;
+    }
+
+    @Override
+    public void returnBook(User user, Book book) {
+        Rent rent = getRent(user, book);
+        rent.setActive(false);
+        sessionFactory.getCurrentSession().update(rent);
+    }
+
+    @Override
+    public Rent getRent(User user, Book book) {
+        Query query = sessionFactory.getCurrentSession()
+                .createQuery("from Rent where user_id LIKE :user_id and book_id LIKE :book_id");
```

Comment:
```suggestion
                .createQuery("from Rent where user_id =:user_id and book_id = :book_id");
```
like використовується для витягування схожих даних. Чи можна використовувати like коли нам треба витягнути запис по id? Якщо ми передамо id= 5, то з бд можуть витягнутись будь-які з наступних записів з  id = 15, 25, 53...